modules/client-configuration/pages/client-proxy-script.adoc

```diff
@@ -0,0 +1,69 @@
+[[script-client-proxy]]
+= Register Clients  to a Proxy with a Bootstrap Script
+
+
+You can register clients through the {productname} Proxy with a bootstrap script.
+This is done almost the same way as registering clients directly with the {productname} Server.
+
+You create the bootstrap script on the {productname} Server with a command line tool.
+The bootstrap script then deploys all necessary information to the clients.
+The bootstrap script requires some parameters such as the FQDN of the proxy, activation keys, or GPG keys.
+These parameters depend on your specific setup, but the FQDN of the proxy is mandatory.
+
+
+
+.Procedure: Registering Clients to a Proxy with a Bootstrap Script
+
+. Create a client activation key on the {productname} Server using the {webui}.
+  For more information, see xref:client-configuration:activation-keys.adoc[].
+. On the container host of the server, start a terminial in the container:
+
++
+
+[source,shell]
+----
+mgrctl term
+----
+
++
+
+--
+.. In the container, to create the bootstrap script, execute the [command]``mgr-bootstrap`` command:
+
++
+
+// FIXME or must we set --http-proxy here?
```

Comment:
`--http-proxy` is when the client should use HTTP proxy to be able to connect to SMLM proxy. There are two different things. So it is not mandatory.